snow/networking/handler/handler_test.go

```diff
@@ -653,3 +682,20 @@ func TestHandlerStartError(t *testing.T) {
 	_, err = handler.AwaitStopped(context.Background())
 	require.NoError(err)
 }
+
+func createSubscriberAndChangeNotifier() (*block.ChangeNotifier, common.Subscription, chan<- common.Message) {
+	var cn block.ChangeNotifier
+
+	messages := make(chan common.Message, 1)
+
+	subscription := func(ctx context.Context) (common.Message, error) {
+		select {
+		case msg := <-messages:
+			return msg, nil
+		case <-ctx.Done():
+			return common.Message(0), ctx.Err()
+		}
+	}
+
+	return &cn, subscription, messages
+}
```

Comment:
Similar comment as to the router, it doesn't seem like the `ChangeNotifier` is related to the `subscription` or the `messages` channel from this function. Can we remove the `ChangeNotifier` from this function?